cibuildwheel/util/resources.py

```diff
@@ -18,6 +19,8 @@
 CIBUILDWHEEL_SCHEMA: Final[Path] = PATH / "cibuildwheel.schema.json"
 
 
+# this value is cached because it's used a lot in unit tests
+@functools.lru_cache
```

Comment:
```suggestion
@functools.cache
```

c.f. https://docs.python.org/3/library/functools.html#functools.cache

> Because it never needs to evict old values, this is smaller and faster than [lru_cache()](https://docs.python.org/3/library/functools.html#functools.lru_cache) with a size limit.